ethical-legal-and-social-issues/index.md

```diff
@@ -1,15 +1,167 @@
+- **Types of data**: One should consider different data types and whether they actually fall under GDPR in an infectious disease study. For instance, clinical-epidemiological data and human sequences are concerned by GDPR, while pathogen genomic sequences and serological data are not subject to GDPR. 
+- **Distributed data**: When dealing with data distributed across multiple countries or outside the original jurisdiction, it's essential to consider the inherent limitations. The location of the data significantly influences the permissions required for access and sharing. Navigating through the complexities of international data laws requires a clear understanding of the regulatory environment in each country involved. Obtaining approval often entails a thorough compliance process with these regulations to facilitate data sharing, ensuring that all data handling practices meet the necessary legal standards.
+- **Legal basis**: When the study deals with personal data, one should consider the legal basis for processing data. Different legal bodies are responsible for components of personal data that fall outside of GDPR. There is no overarching approach at the EU level for public health data (Article 168 TFEU). However, there are specific areas in which the EU Commission or specific institutions have responsibility.
+- **Regulations**: Different funding bodies, institutions, and countries can have different regulations that the researchers must adhere to. It is important to consider checklists and guidelines relevant to one’s particular case. 
+- **Regulatory Evolution**: As regulations and laws are subject to change over time, it is crucial to establish an ongoing monitoring and evaluation mechanism to ensure that chosen strategies remain current and compliant.
+Some data (e.g. genomic data) may be subject to intellectual property, which may restrict the data access and subsequent study and management steps.
+
+### Existing approaches 
+
+- Generally, the principles of ELSI are anchored in four fundamental pillars. These pillars serve as a constant reminder and guide for best practices in human-data-centred research. 
+  - Autonomy – respect for the patient's right to self-determination 
+  - Beneficence – the duty to 'do good' 
+  - Non-maleficence – the duty to 'not do bad' 
+  - Justice – to treat all people equally and equitably 
+- Creating a Data Management Plan that lists all the ELSI bodies that need approval. This can be used as a checklist to determine whether everything that is needed is considered, including how to collect consent. In any case, the institution’s Data Protection - Officer (DPO) is the person to refer to when considering the ethical and legal aspects of data management.
+- Checking the restrictions related to data movement (e.g. between entities, universities, and countries).This can be done by consulting the institution/institutions that hold the data. 
```

Comment:
```suggestion
- Checking the restrictions related to data movement (e.g. between entities, universities, and countries). This can be done by consulting the institution/institutions that hold the data. 
```